Firestore/Example/Tests/Integration/FSTDatastoreTests.mm

```diff
@@ -146,19 +147,19 @@ - (void)rejectListenWithTargetID:(const TargetId)targetID error:(NSError *)error
 
 @end
 
-#pragma mark - FSTDatastoreTests
```

Comment:
FSTDatastoreTests is more like FST+Datastore+Tests (see the filename, FSTDatastoreTests.mm) rather than FSTDatastore+Tests i.e. test FSTDatastore. So maybe you want to keep the FST prefix here since the @interface DatastoreTests : XCTestCase is still an objc class?

Reply:
Done.